### DISCUSSION TITLE
track URL in actived Browser using electron

### DISCUSSION THREAD

**Discussion by 8KCool** [community/community#47846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Thanks...
I am building one software to track employee's action...
I have a real issue to track URL using electron.
You might know about Hufstuff...
I completed all points of this but one point "URL track" is a real issue.
![image](https://user-images.githubusercontent.com/125135541/219828155-233200c8-8eec-48b6-a74b-ae8c0912be6a.png)
I already finished function to get active window, but if the window is browser, I want to know about it's now URL.

**Comment by hrithikdas88** [comment]
i was also looking for the same any updates??
